style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -112,6 +112,36 @@ const metadata: StyleDictionary.MetadataIndex = {
     description:
       'The background color of app layout panel content area. For example: The side navigation and tools panel content background color.',
   },
+  colorBackgroundLayoutToggleActive: {
+    description: 'The background color ot the app layout toggle button when it is active.',
```

Comment:
Suggested rephrase: 'The background color of the app layout toggle button when it's active.'